setup.py

```diff
@@ -25,6 +25,8 @@
     setup_requires=[
         'mock>=1.0.1',
         'testfixtures>=4.1.2',
+        'troposphere>=1.0.0',
```

Comment:
Its inclusion was a mistake, I'll sort and re-PR tomorrow

On 22 June 2015 17:30:31 BST, Ash Berlin <notifications@github.com> wrote:
>> @@ -25,6 +25,8 @@
>>      setup_requires=[
>>          'mock>=1.0.1',
>>          'testfixtures>=4.1.2',
>> +        'troposphere>=1.0.0',
>
>Given troposphere is already in the install_requires list we shouldn't
>need to add it here. What does this fix or what breaks without it?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ministryofjustice/bootstrap-cfn/pull/117/files#r32953939

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.